### ISSUE TITLE
Error when adding samples to bundle

### ISSUE THREAD

**Issue by seallard** [Clinical-Genomics/housekeeper#133]
@fevac 
Has the behaviour for housekeeper changed? I used to be able to store subsampled samples with:

```
housekeeper add bundle $down_sample_id
housekeeper add file -b $down_sample_id -t fastq $down_fastq_r1
housekeeper add file -b $down_sample_id -t fastq $down_fastq_r2
housekeeper include $down_sample_id
```
Now it fails on the second command because the folder is housekeeper is not created.
```FileNotFoundError: [Errno 2] No such file or directory ```

@peterpru 
The command: 
```
housekeeper add file -t fastq -b newlyrapidglider /home/proj/stage/rare-disease/cases/mintkrill/downsampling/HGV2JDMXY_ACC10479A49_S104_L001_R1_001.fastq.gz
```
![image](https://github.com/Clinical-Genomics/housekeeper/assets/33596072/d4273052-c8f2-459d-b1b9-ee9f3f924aea)

**Comment by SofiaOtero** [comment]
The commands work in this order:

```
housekeeper add bundle $down_sample_id
housekeeper include $down_sample_id
housekeeper add file -b $down_sample_id -t fastq $down_fastq_r1
housekeeper add file -b $down_sample_id -t fastq $down_fastq_r2
```
An option could be to include the bundle in the `housekeeper add bundle`command.

**Comment by islean** [comment]
Just a question - was it intended to add a stage file in the production environment?

**Comment by peterpru** [comment]
> Just a question - was it intended to add a stage file in the production environment?

For my code, yes. I had decompressed and downsampled fastq files in stage, which I wanted to save in housekeeper.

Regarding the comment from Sofia, before this error, the commands were the other way around, where include was done last, as also shown in the downsampling page of atlas: https://atlas.scilifelab.se/production/data_analysis/supporting_procedures/downsampling/

**Comment by henrikstranneheim** [comment]
If it is going to be used in production then everything should have been done in the production context. And then you copy the database and the file(s) you created to stage (if you want test something). Avoid doing stage to production as it is bound to cause issue.

**Comment by peterpru** [comment]
> If it is going to be used in production then everything should have been done in the production context. And then you copy the database and the file(s) you created to stage (if you want test something). Avoid doing stage to production as it is bound to cause issues.

I will keep that in mind for the future. 👍  To clarify, the cases were started in production using the files that are in housekeeper, per the instructions in atlas, and not linked directly to files in stage.

**Comment by Vince-janv** [comment]
### Decision:
- Make `houskeeper add bundle` create the housekeeper-bundles/<bundle-name> folder

**Comment by islean** [comment]
Fixed today in #136.
